manifests/v/VSCodium/VSCodium/1.79.2.23166/VSCodium.VSCodium.installer.yaml

```diff
@@ -0,0 +1,40 @@
+# Created with WinGet Releaser v2 using Komac v1.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: VSCodium.VSCodium
+PackageVersion: 1.79.2.23166
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /mergetasks=!runcode
+UpgradeBehavior: install
+Installers:
+- Architecture: x86

```

Comment:
```suggestion
- Architecture: arm64
```

Reply:
Based on the installer name and different hash, I'm guessing this is meant for `User` scope.